### ISSUE TITLE
Question about example_loss_except_other

### ISSUE THREAD

**Issue by dinobby** [wang-h/bert-relation-classification#5]
Hi! really appreciate your nice work and sharing the code. 😄 
Here is a question that just confuses me a little,

```
dist = one_hot_labels[:, 1:].float() * log_probs[:, 1:]
example_loss_except_other, _ = dist.min(dim=-1)
per_example_loss = - example_loss_except_other.mean()
```

why does the min of dist equal the loss without other class here?
thanks for your great work again, looking forward to your reply 😆

**Comment by Hadjer13** [comment]
@wang-h  I have a question regarding the loss used, 
does it exclude data having the label 'Others' ? and why ?
